tests/acs2/testrandom_test.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+from alcs.acs2.testrandom import TestRandom, TestSample
+
+
+class TestRandomTest(unittest.TestCase):
```

Comment:
Better naming, I don't understand the purpose of this test case.

Reply:
`TestRandom` class is for mocking `random()` to test behavior of code in case of different sequences of randoms generated. So, I'm testing it to make sure it can be relied upon when used in other tests. The class itself have to be moved from main to test.

Reply:
Did some renaming to better show the intent of tests.